### DISCUSSION TITLE
2023년 04월 28일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#48] [General]
# 2023년 04월 28일

LOG 네오와 커피챗을 했다. 오늘은 네오와 내내 같이 있을 수 밖에 없었다. 점심에 포이, 망고, 토리와 같이 토도로끼에서 점심을 먹었고, 5시부터는 면담을 했다. 6시에는 회식을 하러 갔으니 오늘 내내 네오와 같이 있었던 셈이다. 면담에서는 페어…

https://cyma-s.github.io/2023-04-28/

**Comment by Choi-JJunho** [comment]
청개구리 베로의 삶을 응원합니다~

**Reply by Cyma-s** [reply]
개굴개굴 🐸

**Comment by drunkenhw** [comment]
아무리 그래도 저처럼 살면 안돼요

**Reply by Cyma-s** [reply]
그게 멋이니까
